queue/src/fiskaltrust.Middleware.Storage.MySQL/Repositories/MySQLActionJournalRepository.cs

```diff
@@ -74,11 +74,17 @@ public async Task<ftActionJournal> GetWithLastTimestampAsync()
             }
         }
 
-        public IAsyncEnumerable<ftActionJournal> GetByPriorityAfterTimestampAsync(int lowerThanPriority, long fromTimestampInclusive)
+        public async IAsyncEnumerable<ftActionJournal> GetByPriorityAfterTimestampAsync(int lowerThanPriority, long fromTimestampInclusive)
         {
+            var query = "SELECT * FROM ftActionJournal WHERE TimeStamp >= @from AND Priority < @prio ORDER BY TimeStamp";
+
             using (var connection = new MySqlConnection(ConnectionString))
             {
-                return connection.Query<ftActionJournal>($"SELECT * FROM ftActionJournal WHERE TimeStamp >= @from AND Priority < @prio ORDER BY TimeStamp", new { from = fromTimestampInclusive, prio = lowerThanPriority }, buffered: false).ToAsyncEnumerable();
+                await connection.OpenAsync().ConfigureAwait(false);
```

Comment:
How was this working in the past?

Reply:
It did not work but the method is only used by the FR queue in specific cases and we had no acceptance test for this method.